mne/epochs.py

```diff
@@ -3305,9 +3161,16 @@ def _read_one_epoch_file(f, tree, preload):
         if drop_log is None:
             drop_log = ((),) * len(events)
 
+    # as of v1.0, Epochs support Annotations which require raw_sfreq
+    # to be stored. However, for backwards compatibility if raw_sfreq
+    # is None (not found) in the file, then we will default raw_sfreq
+    # # to info['sfreq']
+    if raw_sfreq is None:
+        raw_sfreq = info['sfreq']
```

Comment:
Do you actually need to do this here? Even if it's `None`, the `BaseEpochs` should be able to use `raw_sfreq=None` as this alias already.

Reply:
Removed.